### PR TITLE
[Feature] Add a http interface to acquire the tablets distribution between different disks of a particular BE

### PR DESCRIPTION
**Pull request by weizuo93**
## Proposed changes

For the task of rebalancing tablet among different disks on the same BE, I think it might be an effective strategy to ensure all tablets under the same partition evenly distribute on the different disks. Thus, it is necessary to obtain the distribution of tablets under the same partition between different disks on a BE. 

This patch  add a new http interface for FE to acquire the distribution of tablets under each partition between different disks of a particular BE.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)
- [] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [] I have create an issue on (Fix #ISSUE), and have described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [] I have added tests that prove my fix is effective or that my feature works
- [x] If this change need a document change, I have updated the document
- [x] Any dependent changes have been merged

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/http/rest/TabletDiskDistributionAction.java [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+        jsonObject_result.put("code", 0);
+        jsonObject_result.put("data", jsonObject_backend);
+        jsonObject_result.put("count", tablet_total);
+        String result =jsonObject_result.toString();
+
+        response.setContentType("application/json");
+        response.getContent().append(result);
+        sendResult(request, response);
+    }
+}
```
Comment (acelyc111) [inline fe/fe-core/src/main/java/org/apache/doris/http/rest/TabletDiskDistributionAction.java:192]: Add a newline

File: fe/fe-core/src/main/java/org/apache/doris/http/rest/TabletDiskDistributionAction.java [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+                                        tablet_total++;
+                                    }
+                                }
+                            }
+                        }
+                        partitionToTablet.put(partitionId, tabletSchemaHash);
+                    }
+                }
+            } catch (Exception e) {
+                LOG.info(e);
```
Comment (acelyc111) [inline fe/fe-core/src/main/java/org/apache/doris/http/rest/TabletDiskDistributionAction.java:133]: When exception throwed, an error response should be returned.

File: fe/fe-core/src/main/java/org/apache/doris/http/rest/TabletDiskDistributionAction.java [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+            } finally {
+                db.readUnlock();
+            }
+        }
+
+        JSONObject jsonObject_backend = new JSONObject();
```
Comment (acelyc111) [inline fe/fe-core/src/main/java/org/apache/doris/http/rest/TabletDiskDistributionAction.java:139]: Naming `jsonObjectBackend ` or `json_object_backend `

Comment (acelyc111) [inline fe/fe-core/src/main/java/org/apache/doris/http/rest/TabletDiskDistributionAction.java:139]: How about split this function to two parts? One for collecting info, the other for construct JSON?

Comment (weizuo93) [reply fe/fe-core/src/main/java/org/apache/doris/http/rest/TabletDiskDistributionAction.java]: OK

**Pull request closed without merging**